pyproject.toml

```diff
@@ -1,11 +1,12 @@
 [tool.poetry]
 name = "uw-idp-web-tests"
 version = "0.1.0"
+package-mode = false
 description = ""
 authors = ["Tom Thorogood <goodtom@uw.edu>", "Glen Goulter <glenrg@uw.edu>"]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.10"
```

Comment:
3.10 basically b/c we dont want 3.8 to be installed. 3.10 or higher, excluding 4.0.0 and above.